shared/cfg/base.cfg

```diff
@@ -129,22 +129,22 @@ q35, arm64-pci, arm64-mmio:
 # Serial port support
 # You can assign more than one serial to guest with this parameter.
 # like:
-# serials = "S1 S2 S3"
-serials = "serial0"
-
-# virtio_serialport and virtio_console ports
-# virtio_ports = "vs1 vs2 vc1"
-# Port types
-# virtio_port_type = serialport
-# virtio_port_type_vc1 = console
-# Host socket type
-# virtio_port_chardev = spicevmc
-# virtio_port_chardev_vc1 = socket
-# chardev name prefix (port index will be appended)
+# serials = "serial0 vs1 vs2 vc1"
+# serial_type_vs1 = virtserialport
+# serial_type_vc1 = virtconsole
+# Chardev backend type: 'unix_socket' by default
+# serial_chardev_backend_vs1 = unix_socket
+# serial_chardev_backend_vc1 = tcp_socket
+# Chardev host for tcp/udp socket backend
+# serial_chardev_host = $host_ip (127.0.0.1 by default)
+# Virtio port name prefix (port index will be appended)
 # virtio_port_name_prefix = "com.redhat.spice."
 # By default the port name is used (vc1)
 # virtio_port_name_prefix_vc1 = ""
 
+serials = "serial0"
+serial_type = "isa-serial"
```

Comment:
To avoid  other operation in case level affect the default serial session by mistake,  how do you think about use fixed `serial_type_serial0 = isa-serial` ?  
And in qemu_vm,  use `self.serial_session_device = serial0` instead of `serials[0]`
```
        if serials:
            self.serial_session_device = serials[0]
```